src/view/boot.ts

```diff
@@ -18,6 +18,7 @@ import {MdIconRegistry} from '@angular2-material/icon/icon';
 import {CoreWebService} from "../api/services/core-web-service";
 import {LoginService} from "../api/services/login-service";
 import {RoutingService} from "../api/services/routing-service";
+import {FormatDate} from "../api/services/format-date-service";
```

Comment:
not really, we just need to inject from the beginning the statefull service, and according what we talked this morning we shouldn't have many (or any at all) statefull service